[feature][KLD-215]: configure app with `dex-config.json` (#202)
soramitsu/klaytn-dex-frontend: docs/deploy.md

  npm i -g pnpm
  ```

> ## Configuration

Author: @outoftardis, please take a look.

> App configuration is split into 2 parts: main configuration file and environment variables.

Reviewer: ```suggestion
App configuration is split into two parts: main configuration file and environment variables.
```

### Main configuration file

> Most of application configuration should be defined in `dex-config.json` file. **This configuration is mandatory.** Also, its contents is strictly validated before being passed into application.

Reviewer: ```suggestion
Most of application configuration should be defined in `dex-config.json` file. **This configuration is mandatory.** Its content is strictly validated before being passed into application.
```

> The main configuration includes:

Reviewer: ```suggestion
The main configuration in `dex-config.json` includes:
```

> - tokens whitelist + native and dex token

Reviewer: ```suggestion
- native and DEX tokens
- whitelisted tokens
```

- network configuration
- addresses of smart contracts deployed in the network
- URIs of subgraphs

> `dex-config.example.json` contains a sample configuration, also used by Soramitsu in dev environment.

Reviewer: ```suggestion
The `dex-config.example.json` file contains a sample configuration used by Soramitsu in dev environment.
```

> For more details of configuration schema please consult [`etc/vite-plugin-config.ts`](../etc/vite-plugin-config.ts).

Reviewer: ```suggestion
For more details of configuration schema, consult [`etc/vite-plugin-config.ts`](../etc/vite-plugin-config.ts).
```
